### ISSUE TITLE
support itunes

### ISSUE THREAD

**Issue by facundoolano** [facundoolano/google-play-api#3]
consider using this same api to expose bothe google play and itunes app (via app-store-scraper)

**Comment by sgargbugreporter** [comment]
Great idea! Would need a way to differentiate between the two. How are you thinking? Same thing but on different ports?

**Comment by facundoolano** [comment]
Probably different paths of the same app

**Comment by NT-ZG** [comment]
This would be a great feature. Any plans to implement it?

**Comment by facundoolano** [comment]
Not really, sorry, I currently have little time for open source, and other projects have higher priority than this. Pull requests are welcome, though.

**Comment by NT-ZG** [comment]
Fair enough, thanks for the quick response.
